src/views/home.hbs

```diff
@@ -0,0 +1,74 @@
+
+    <!-- Showcase section -->
+    <div class="showcase">
+        <div class="container">
+            <div class="content">
+                <h1>Chat It Out </h1>
+                <h2>Gaza Psychological Service</h2>
+                <p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa.
+                    Cum sociis natoque penatibus et magnis
+                </p>
+                <button type="submit">Join Us</button>
+            </div>
+        </div>
+    </div>
+    <!-- Thereby conculdes the showcase section -->
+
+    <!-- Our services & contact us sections -->
+    <main>
+        <section>
+            <h1>Our Services</h1>
+            <div class="container2">
```

Comment:
try to avoid using class name like container 1,2,3...

Reply:
Ookay